iOS_SDK/OneSignalSDK/UnitTests/Shadows/OneSignalClientOverrider.m

```diff
@@ -189,12 +189,28 @@ + (void)finishExecutingRequest:(OneSignalRequest *)request onSuccess:(OSResultSu
         [self didCompleteRequest:request];
 
         if (successBlock) {
-            if ([request isKindOfClass:[OSRequestGetIosParams class]])
+            if ([request isKindOfClass:[OSRequestGetIosParams class]]) {
                 successBlock(self.iosParamsResponse);
-            else if (mockResponses[NSStringFromClass([request class])])
+            }
+            else if (mockResponses[NSStringFromClass([request class])]) {
                 successBlock(mockResponses[NSStringFromClass([request class])]);
-            else
-                successBlock(@{@"id": @"1234"});
+            }
+            else {
+                successBlock(@{
+                @"success" : @(true),
+                @"id" : @"1234",
+                @"in_app_messages" : @[
```

Comment:
won't this affect other test cases?

Reply:
Everything passed fine for me, adding extra expected params to on_sesison response shouldn't cause any issues